README.md

```diff
@@ -8,16 +8,18 @@ love to talk to you in [#knative-ux](https://cloud-native.slack.com/messages/kna
 | Document Folder            | [Folder](https://drive.google.com/drive/folders/1XzZGqV7yHo38d_l7rH1uSIrbQp3JlbBP?usp=sharing)                                                          |
-| Slack Channel              | [#knative-ux](https://cloud-native.slack.com/messages/knative-ux)                                                                                  |
-| Github Repository          | [knative/ux](https://github.com/knative/ux)                                                                                                       |
+| Slack Channel              | [#knative-ux](https://cloud-native.slack.com/messages/knative-ux) (need to join [CNCF Slack](https://slack.cncf.io/) for the first time)                         |
+| Github Repository          | [/ux](https://github.com/knative/ux)                                                                                                                             |                                                                                                  |
 | Github Team WG leads       | [@knative/ux-wg-leads](https://github.com/orgs/knative/teams/ux-wg-leads/members)                                                                       |
 
-| &nbsp;                                                        | Leads           | Company | Profile                                             |
-| ------------------------------------------------------------- | --------------- | ------- | --------------------------------------------------- |
-| <img width="30px" src="https://github.com/omerbensaadon.png"> | Omer Bensaadon  | VMware  | [omerbensaadon](https://github.com/omerbensaadon)   |
-| <img width="30px" src="https://github.com/csantanapr.png">    | Carlos Santana  | IBM     | [csantanapr](https://github.com/csantanapr)         |
+| &nbsp;                                                               | Leads            | Company          | Profile                                                  |
+| -------------------------------------------------------------------- | ---------------- | ---------------- | -------------------------------------------------------- |
+| <img width="30px" src="https://github.com/cali0707.png">             | Calum Murray     | Red Hat          | [cali0707](https://github.com/cali0707)                  |
+| <img width="30px" src="https://github.com/zainabhusain227.png">      | Zainab Husain    | OCAD University  | [zainabhusain227](https://github.com/zainabhusain227)    |
+| <img width="30px" src="https://github.com/mmejia02.png">             | Mariana Mejia    | OCAD University  | [mmejia02](https://github.com/mmejia02)                  |
+| <img width="30px" src="https://github.com/Leo6Leo.png">              | Leo Li           | Red Hat          | [Leo6Leo](https://github.com/Leo6Leo)                     |
```

Comment:
@Leo6Leo since we have different types of leads (product, technical, design) could you maybe add another column to this table reflecting those roles?